distributed-commandbus-springcloud/src/test/java/org/axonframework/springcloud/commandhandling/SpringCloudCommandRouterTest.java

```diff
@@ -387,18 +441,18 @@ public void testUpdateMembershipsOnHeartbeatEventTwoInstancesOnSameServiceIdUpda
     }
 
     private void assertMember(String expectedMemberName, URI expectedEndpoint, boolean localMember,
```

Comment:
I would remove first two parameters since they are always set to `SERVICE_INSTANCE_ID` and `SERVICE_INSTANCE_URI` respectively.